koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -1308,6 +1310,8 @@ def populate_vm_tag_based_costs(self, start_date, end_date, provider_uuid, tag_b
 
             sql = pkgutil.get_data("masu.database", file_path).decode("utf-8")
             for sql_params in param_list:
+                if "hourly_" in file_path:
```

Comment:
I am making some structural changes to this area my [pull request](https://github.com/project-koku/koku/pull/5673/files) that will make it easier to [add metric specific parameters] to this flow.